packages/jest-runtime/src/index.ts

```diff
@@ -1366,13 +1366,11 @@ class Runtime {
       });
     })();
 
-    const path = this._path;
+    const testPath = this._testPath;
```

Comment:
we don't need this, just do `this._testPath` below

Reply:
Got it